spec/services/applications/accept_spec.rb

```diff
@@ -19,8 +19,8 @@
     let(:course_group) { create(:course_group, name: "leadership") }
     let(:course) { create(:course, :senior_leadership, course_group:) }
     let(:lead_provider) { create(:lead_provider) }
-    let(:cohort) { create(:cohort, :current) }
-    let(:cohort_next) { create(:cohort, :next) }
+    let(:cohort) { create(:cohort, :current, funding_cap: false) }
```

Comment:
should the default be false 🤔 why did this start failing as it was ok before?

Reply:
Reverted this, should pass now